Documentation/Functions/Typolink.rst

```diff
@@ -543,27 +543,30 @@ userFunc
     :name: typolink-userFunc
     :type: :ref:`data-type-function-name`
 
-    This passes the link-data compiled by the typolink function to a user-
-    defined function for final manipulation.
+    This passes the link-data compiled by the :typoscript:`typolink` function
+    to a user-defined PHP function for final manipulation.
 
     The :php:`$content` variable passed to the user-function (first parameter) is
     an array with the keys "TYPE", "TAG", "url", "targetParams" and
     "aTagParams".
 
-    TYPE is an indication of link-kind: mailto, url, file, page
+    `TYPE` is an indication of link-kind: `mailto`, `url`, `file`, `page`
 
-    TAG is the full <A>-tag as generated and ready from the typolink
+    `TAG` is the full `<a>`-tag as generated and processed from the :typoscript:`typolink`
```

Comment:
There is no TAG anymore. There is a LinkResult containing all information to build a tag